common/test/opentelemetry/common/request_attributes_test.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::Common::HTTP::RequestAttributes do
+  subject { Class.new.extend(OpenTelemetry::Common::HTTP::RequestAttributes) }
+
+  it 'returns http attributes matching the spec given input' do
+    attributes = subject.from_request(method: 'GET', config: {}, uri: URI('http://example.com/foo?bar=baz'))
+    _(attributes).must_equal(
+      'http.method' => 'GET',
+      'http.scheme' => 'http',
+      'http.url' => 'http://example.com/foo?bar=baz',
+      'http.target' => '/foo?bar=baz',
+      'net.peer.name' => 'example.com',
+      'net.peer.port' => 80
+    )
+  end
+
+  it 'returns http attributes matching the spec given input' do
```

Comment:
Is this identical to the above example or have I not had enough coffee?

Reply:
it was i who needed more coffee

Reply:
updated (removed dupe). good catch